### DISCUSSION TITLE
When to support YouTube short?

### DISCUSSION THREAD

**Discussion by joeycih** [Tyrrrz/YoutubeExplode#748] [General]
Thank you

**Comment by freemanlovelace** [comment]
Good topic.
I was working on that, but unfortunatly, my request client does'nt works anymore or am missing the right configuration. Do you have a working client ?

**Comment by Tyrrrz** [comment]
Shorts are already supported

**Comment by Laiteux** [comment]
@Tyrrrz what about a `IsShort` bool? There seems to be no reliable way of detecting if a video is a short or not, or is there?

Checking for `/shorts/` in URL is unreliable as links can be `/watch` ones.
